src/OpenSage.Game/Graphics/Cameras/TacticalView.cs

```diff
@@ -607,7 +607,7 @@ internal void RotateCameraTowardPosition(Vector3 pos, double milliseconds, float
             angle = -angle;
         }
 
-        // Default camera is rotated 90 degrees, so match
+        // NoArmor camera is rotated 90 degrees, so match
```

Comment:
I won't comment all the places this happened, but looks like a refactor maybe went a bit awry?

Reply:
Thanks for spotting that. I let Visual Studio look in comments when renaming a property, but that was a bad idea in this case...